src/v2i-hub/TelematicBridgePlugin/manifest.json

```diff
@@ -11,6 +11,31 @@
             "key": "LogLevel",
             "default": "INFO",
             "description": "The log level for this plugin"
+        },
+        {
+            "key": "NATSUrl",
+            "default": "nats://127.0.0.1:4222",
+            "description": "The log level for this plugin"            
```

Comment:
Incorrect descriptions for all configuration parameters

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/567/commits/e8678760c97cbb7d5bd212ac07f490793d7d8c63